lambda/update/certificate_provider_sign_test.go

```diff
@@ -61,7 +69,124 @@ func TestValidateUpdateCertificateProviderSign(t *testing.T) {
 					},
 					{
 						Key: "/certificateProvider/signedAt",
-						New: json.RawMessage(`"` + time.Now().Format(time.RFC3339) + `"`),
+						New: json.RawMessage(`"` + now.Format(RFC3339local) + `"`),
+						Old: jsonNull,
+					},
+					{
+						Key: "/certificateProvider/contactLanguagePreference",
+						New: json.RawMessage(`"cy"`),
+						Old: jsonNull,
+					},
+					{
+						Key: "/certificateProvider/email",
+						New: json.RawMessage(`"b@example.com"`),
+						Old: jsonNull,
+					},
+					{
+						Key: "/certificateProvider/channel",
+						New: json.RawMessage(`"online"`),
+						Old: jsonNull,
+					},
+				},
+			},
+		},
+		"valid - with previous values": {
```

Comment:
This doesn't prove anything right now but once consumer tests are updated (and the mock LPA returned set to have the expected existing values) it will cover old == existing.

Reply:
It's a bit of a weird one as the consumer gets the existing LPA values from the [mock apigateway](https://github.com/ministryofjustice/opg-data-lpa-store/blob/MLPAB-1546-support-channel-change/mock-apigw/main.go#L117) that won't match the MRLPA tests that are in place at the moment and I can't make this change until the consumer tests pass.

So I'm planning on merging this with the new changes, updating the MRLPA consumer tests to have existing values that match mock apigateway along with sending channel and email and then making a new PR here that drops the option for MustMatchExisting. It's a bit convoluted but I was struggling to get around the existing LPAS values being defined in mock apigateway.

Reply:
fair enough